src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -39,17 +34,7 @@ function CompareResultsTable(props: CompareResultsProps) {
             ))}
           </TableRow>
         </TableHead>
-        <TableBody>
-          {compareResults.length > 0 &&
-            compareResults.map((result, index) => (
-              <CompareResultsTableRow
-                key={index}
-                result={result}
-                index={index}
-                mode={mode}
-              />
-            ))}
-        </TableBody>
+        <PaginatedCompareResults mode={mode} />
```

Comment:
Initially the design included the pagination as an extra feature that can be enabled or disabled by users. Is this still the case?

Reply:
There is an option in the 'results per row' to include 'All'. does this suffice?